pkg/kromgo/kromgo.go

```diff
@@ -14,13 +16,44 @@ import (
 	"go.uber.org/zap"
 )
 
-func KromgoRequestHandler(w http.ResponseWriter, r *http.Request, config configuration.KromgoConfig) {
+type KromgoHandler struct {
+	Config         configuration.KromgoConfig
+	BadgeGenerator *badge.Generator
+}
+
+// NewKromgoHandler initializes the handler with the necessary dependencies
+func NewKromgoHandler(config configuration.KromgoConfig) (*KromgoHandler, error) {
+	var badgeGenerator *badge.Generator
+	if config.Badge.Font != "" {
```

Comment:
I was talking with onedr0p about this, we were thinking that since the font is already included in the container, we should automatically set it. I can add this after merging.